djangofromscratch/JWTLogin/views.py

```diff
@@ -6,10 +7,21 @@
 from .serializers import UserSerializer
 
 
+def send_email(user):
```

Comment:
Is it possible to define this function with message as variable so that we don't need to write it everytime for different kinds of email?

Reply:
Yes, I will correct it in new PR

Reply:
why not in this pr?